azure-stack-go-sdk/azure-stack-go-sdk-profiles.md

```diff
@@ -0,0 +1,302 @@
+---
+title: Installing and use Azure Go SDK profiles on Azure Stack
+description: How to install and use Azure Go SDK profiles on Azure Stack environment.
+keywords: azure, azure stack, sdk, go, golang, azure sdk
+author: seyadava
+ms.author: seyadava
+ms.date: 03/22/2018
+ms.topic: article
+ms.devlang: go
+manager: knithinc
+---
+
+# What is a profile?
+A profile is a combination of different resource types with different versions from different services. Using a profile will help you mix and match between various resource types. Profiles can provide:
+
+* Stability for your application by locking to specific API versions; and/or
```

Comment:
Do not use bulleted lists in this style. These elements should be part of the sentence `Profiles can provide`.